ethwrite/src/contract_helpers.ts

```diff
@@ -0,0 +1,37 @@
+import { ethers } from 'ethers'
```

Comment:
As this is only used in testing, send it there (under `/test/helpers` should be fine)